docs/FAQ.md

```diff
@@ -47,9 +47,9 @@ To use the `Microsoft.PowerShell.SDK` NuGet package, declare `PackageReference`
 
 ```xml
 <ItemGroup>
-  <PackageReference Include="Microsoft.PowerShell.SDK" Version="6.0.0-beta.9" />
-  <PackageReference Include="Microsoft.PowerShell.Commands.Diagnostics" Version="6.0.0-beta.9" />
-  <PackageReference Include="Microsoft.WSMan.Management" Version="6.0.0-beta.9"/>
+  <PackageReference Include="Microsoft.PowerShell.SDK" Version="6.0.0-rc" />
+  <PackageReference Include="Microsoft.PowerShell.Commands.Diagnostics" Version="6.0.0-rc" />
+  <PackageReference Include="Microsoft.WSMan.Management" Version="6.0.0-rc"/>
```

Comment:
Don't need to update example content.

Reply:
People typically copy-paste this.. I believe we should point to the latest. Your thoughts?

Reply:
There are many other examples in our docs or scripts, and we don't update the version numbers in them, for example, `6.0.0-alpha.9` in https://github.com/PowerShell/PowerShell/blob/master/docs/installation/windows.md.